versions/3.0.4.md

```diff
@@ -1058,7 +1058,7 @@ simple | `primitive`, `array`, `object` | `path`, `header` | Simple style parame
 form | `primitive`, `array`, `object` | `query`, `cookie` | Form style parameters defined by [RFC6570](https://tools.ietf.org/html/rfc6570#section-3.2.8). This option replaces `collectionFormat` with a `csv` (when `explode` is false) or `multi` (when `explode` is true) value from OpenAPI 2.0.
 spaceDelimited | `array`, `object` | `query` | Space separated array values or object properties and values. This option replaces `collectionFormat` equal to `ssv` from OpenAPI 2.0.
 pipeDelimited | `array`, `object` | `query` | Pipe separated array values or object properties and values. This option replaces `collectionFormat` equal to `pipes` from OpenAPI 2.0.
-deepObject | `object` | `query` | Provides a simple way of rendering nested objects using form parameters.
+deepObject | `object` | `query` | Allows objects with scalar properties to be represented using form parameters. The representation of array or object properties is not defined.
```

Comment:
Looking at the examples in the next section and the "type" column shouldn't this be
```suggestion
deepObject | `object` | `query` | Allows objects with scalar properties to be represented using form parameters. The representation of primitive or array values is not defined.
```